proposals/p2922.md

```diff
@@ -0,0 +1,153 @@
+    community (Carbon leads), and our current framework does not allow for clear
+    accountability in terms of their own conduct due to this lack of separation.
+-   The diversity of backgrounds and perspectives is limited among our current
+    CoC team.
+-   The Carbon leads have limited time and availability to intervene in urgent
+    and important misconduct situations.
+
+So, it is time to build a specialized team to deal with escalations which is not
+subordinate to the Carbon leads, and is able to handle diverse misconduct
+escalations in an adequate and timely manner. This means we can recruit and/or
+train people from diverse backgrounds to handle such escalations for the Carbon
+community, and provide competent support to the leads, the rest of the
+moderation team, and everyone else in the community.
+
+TODO: What problem are you trying to solve? How important is that problem? Who
+is impacted by it?
+
+## Background
+
+Our Code of Conduct:
+https://github.com/carbon-language/carbon-lang/blob/trunk/CODE_OF_CONDUCT.md
+
+Our current CoC team:
+
+-   Chandler Carruth (@chandlerc on Discord and GitHub)
+-   Kate Gregory (@KateGregory on Discord and GitHub)
+-   Richard Smith (@zygoloid on Discord and GitHub)
+
+## Proposal
+
+Recruit and train 5+ potential new conduct team members to deal with
+conduct-related escalations on behalf of and for the Carbon community, and
+replace the current CoC team as soon as enough members are ready.
+
+We should always have at least 3 conduct team members available to respond
+promptly, even factoring in vacations, getting sick, or other normal
+disruptions.
+
+Subsequent updates to the exact members will be managed by the new conduct team
+themselves, and without full proposals but just PRs to update the relevant
+documentation.
+
+## Details
+
+### Initial team members
+
+-   TBA
+
+We will continue to look for more candidates in order to get to a point where we
+have a sustainable number, especially as the community grows.
+
+### Process
+
+Selection and sourcing:
+
+-   Candidates were identified and recruited by our Community lead.
+-   Candidates met with both Carbon leads and the Community lead individually to
+    make sure everyone was comfortable with the role and responsibilities
+    involved.
+
+Onboarding and support:
+
+-   New members will have access to coaching and mentoring from the Community
+    lead as a diversity, equity and inclusion specialist, and general support by
+    a senior moderation team member.
+-   We organize regular informal moderation meetups to learn with and from each
+    other, including a book club.
+
+Exit:
+
+-   We are aware that this work tends to be straining and that each person's
+    involvement may only be temporary.
+-   We will make sure that conduct team members feel safe to leave the team and
+    community when they need to step out by developing a bench of candidates.
```

Comment:
This specific case is talking about the proposers and the current conduct team. Proposal docs often use "we" I think? If there is a better wording, happy to update!

Reply:
This is a forward-looking, ongoing commitment (unlike most uses of "we" in proposals), so it seems important to be clear about who is responsible for following through on it.

> This specific case is talking about the proposers and the current conduct team.

Since this is an ongoing responsibility, it seems like it would be better to vest it in an ongoing body like the leads or the conduct team, rather than ~4 specific people. It also seems like the conduct team ought to have a role in developing the bench, since they'll be the ones selecting people from it.

Reply:
Ok, I'm not sure I fully understand the concern here, but I've reworded these sections to hopefully address it.

Reply:
This looks good, thanks.